os/fs/romfs/fs_romfs.c

```diff
@@ -1040,6 +1102,47 @@ static int romfs_statfs(FAR struct inode *mountpt, FAR struct statfs *buf)
 }
 
 /****************************************************************************
+ * Name: romfs_stat_common
+ *
+ * Description:
+ *   Return information about a dentry - directory entry: file or directory
+ *   Dentry is described by:
+ *    * type (file or directory, access rights)
+ *    * size
+ *    * sectorsize (filesystem global)
+ *  Resulting information is written into 'buf'
+ *
+ ****************************************************************************/
+
+static int romfs_stat_common(uint8_t type, uint32_t size, uint16_t sectorsize, FAR struct stat *buf)
+{
+	memset(buf, 0, sizeof(struct stat));
+	if (IS_DIRECTORY(type)) {
+		/* It's a read-execute directory name */
+		buf->st_mode = S_IFDIR | S_IROTH | S_IXOTH | S_IRGRP | S_IXGRP |
+					   S_IRUSR | S_IXUSR;
+	} else if (IS_FILE(type)) {
+		/* It's a read-only file name */
+		buf->st_mode = S_IFREG | S_IROTH | S_IRGRP | S_IRUSR;
+		if (IS_EXECUTABLE(type)) {
+			/* It's a read-execute file name */
+
+			buf->st_mode |= S_IXOTH | S_IXGRP | S_IXUSR;
+		}
+	} else {
+		/* Otherwise, pretend like the unsupported type does not exist */
+		fvdbg("Unsupported type: %d\n", type);
```

Comment:
It is error case, so fdbg is right here because we have only 2 types.

Reply:
Done.